.vscode/tasks.json

```diff
@@ -2,45 +2,67 @@
     "version": "2.0.0",
     "tasks": [
         {
-            "label": "make",
+            "label": "Run dev-tools image (on Windows with PowerShell)",
             "type": "shell",
-            "command": "make clean && make",
+            // Checks if a container with name ${config:container_name} exists, otherwise creates it in detached mode with the proper parameters.
```

Comment:
tasks.json is not a "true" json file as vscode accepts comments. So no worries if github highlights that as an error.